internal/appsec/remoteconfig.go

```diff
@@ -17,6 +17,8 @@ import (
 	rc "github.com/DataDog/datadog-agent/pkg/remoteconfig/state"
 )
 
+const envRules = "DD_APPSEC_RULES"
```

Comment:
Maybe worth exposing them out of appsec-internal-go, like DD_APPSEC_ENABLED.

Reply:
DD_APPSEC_ENABLED is specific to this repo so I'm not sure it would be that relevant to export it from internal-go (it's not even in there to begin with). We can export it in this one though, sure: 063dc06b497df

CFG env & values export PR: https://github.com/DataDog/appsec-internal-go/pull/16